massgov/mayflower-patternlab
# DP-5480 Rename Patterns

## This PR contains changes for the 5.8 release.  Do not merge until that release has been made.

## Description
Rename patterns to make them more appropriate for their current use.  Several patterns were built for a specific use and then later adapted to be used in other places.  As such the original name of the pattern no longer properly explains the use of the pattern.

These are the files that were renamed:
Action Finder => Featured Callouts
Sections Three Up => Three Up Cards
Section Links => Topical Card
Emergnecy Alerts => Site Alerts
Column Heading => Content Heading
Single Column Template => Full Width Template
Two Column Reverse => Left Rail Template
Two Column => Right Rail Template
Emergency Header => Alert Header
Emergency Alert => Alert Message
Location Icons => Labelled Icons
Key Actions => Callout Grid
Cat icon => Category Icon
Callout time => Icon Callout
Event Listing => Event List

## Related Issue / Ticket
https://jira.state.ma.us/browse/DP-5480

## Steps to Test
1. Browse through Mayflower to see that the new names are there and that the old names aren't
2. In Mayflower compare each page between your local and production to make sure all the patterns are still rendering correctly.
